codec-http2/src/main/java/io/netty/handler/codec/http2/Http2StreamFrame.java

```diff
@@ -19,24 +19,24 @@
 
 /**
  * A frame whose meaning <em>may</em> apply to a particular stream, instead of the entire
- * connection. It is still possibly for this frame type to apply to the entire connection. In such
- * cases, the {@code stream} reference should be {@code null} or an object referring to the
- * connection.
- *
- * <p>The meaning of {@code stream} is context-dependent and may change as a frame is processed in
- * the pipeline.
+ * connection. It is still possible for this frame type to apply to the entire connection. In such
+ * cases, the {@link #streamId()} must return {@code 0}. If the frame applies to a stream, the
+ * {@link #streamId()} must be greater than zero.
  */
 @UnstableApi
 public interface Http2StreamFrame extends Http2Frame {
+
     /**
-     * Set the stream identifier for this message.
-     *
-     * @return {@code this}
```

Comment:
The return documentation is helpful to know what you get back. Otherwise you have to guess.